script.js

```diff
@@ -6,16 +6,13 @@ function opened (){
     menuOpened.style.display = "block";
 }
 
-menu.addEventListener("click", opened);
 menu.addEventListener("mouseover", opened);
+menu.addEventListener("mouseleave", esconderMenu);
```

Comment:
Adicionando um fix para esconder o menu quando o mouse for removido. Isso dará alguns problemas que veremos depois, principalmente quando passarmos o mouse fora por alguns instantes, mas já dá pra ver o menu fechando e clicar nos elementos se for direto pra box